10일차/키패드_누르기.js

```diff
@@ -0,0 +1,60 @@
+function solution(numbers, hand) {
+    let answer = '';
+    let leftHand = '*';
+    let rightHand = '#';
+
+    const getDistance = (handPosition, number) => { // 거리 구하는 함수
+        const keypad = { // 키패드 좌표
+            1: [0, 0], 2: [0, 1], 3: [0, 2],
+            4: [1, 0], 5: [1, 1], 6: [1, 2],
+            7: [2, 0], 8: [2, 1], 9: [2, 2],
+            '*': [3, 0], 0: [3, 1], '#': [3, 2]
+        };
+        const [x1, y1] = keypad[handPosition]; // 현재 손 위치
+        const [x2, y2] = keypad[number]; // 누를 번호 위치
+        return Math.abs(x1 - x2) + Math.abs(y1 - y2);
+    };
+
+    const isLeftHand = (number) => [1, 4, 7].includes(number); // 왼손으로 누를 번호
+    const isRightHand = (number) => [3, 6, 9].includes(number); // 오른손으로 누를 번호 
```

Comment:
```suggestion
    const checkIsLeftHand = (number) => [1, 4, 7].includes(number); // 왼손으로 누를 번호
    const checkIsRightHand = (number) => [3, 6, 9].includes(number); // 오른손으로 누를 번호 
```
함수는 동사로 시작하는게 좋아요.
[자바스크립트 스타일 가이드](https://velog.io/@cada/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8-%EC%8A%A4%ED%83%80%EC%9D%BC-%EA%B0%80%EC%9D%B4%EB%93%9C-%EB%84%A4%EC%9D%B4%EB%B0%8D-%EC%BB%A8%EB%B2%A4%EC%85%98-%ED%8E%B8#%ED%95%A8%EC%88%98-)